src/WebJobs.Extensions/Extensions/Timers/Scheduling/ScheduleMonitor.cs

```diff
@@ -14,7 +14,12 @@ namespace Microsoft.Azure.WebJobs.Extensions.Timers
     /// </summary>
     public abstract class ScheduleMonitor
     {
-        private static DateTime defaultDateTime = new DateTime(0, DateTimeKind.Local);
+        // Avoid anything close to DateTime.MinValue as time zone conversions have caused bugs
+        internal static readonly DateTime DefaultDateTime = default(DateTime).ToLocalTime().AddYears(1);
```

Comment:
Do we know if this would cause any breaking changes somewhere? Does anywhere in Functions/Antares depend on the MinValue ?

Reply:
Good callout. We went back to MinValue just to be safe.